UnitTests/BraintreeApplePayTests/BTApplePay_Tests.swift

```diff
@@ -271,7 +271,7 @@ class BTApplePay_Tests: XCTestCase {
         }
         waitForExpectations(timeout: 2)
         
-        XCTAssertTrue(mockClient.postedAnalyticsEvents.contains("ios.apple-pay.network-connection.failure"))
+        XCTAssertTrue(mockClient.postedAnalyticsEvents.contains("apple-pay:tokenize:network-connection:failed"))
```

Comment:
Lol this lonely unit test reveals how much we aren't unit testing analytics. Shall we make a JIRA for this as a 🏈 punt ?

Another thing - we sort of inconsistently test analytics. Sometimes we have separate tests, other times we bake analytic event checks into existing tests. Would be nice to standardize our testing strategy.

Reply:
😢 yeah I noticed that too. We should come up with a strategy then audit all of the tests. Having all of the analytics in 1 file will have the advantage of being able to quickly see test coverage for out events, which will be nice! Spike ticket to determine our pattern is: DTBTSDK-2441